sklearn/kernel_approximation.py

```diff
@@ -7,7 +7,6 @@
 #         Daniel Lopez-Sanchez (TensorSketch) <lope@usal.es>
 
 # License: BSD 3 clause
-
 import warnings
```

Comment:
Can you undo the deletion of this line? We try to minimize non-functional changes such as these to better preserve `git blame` history.